examples/agentInteractionsDemo/util.js

```diff
@@ -0,0 +1,120 @@
+// ----- Utility Functions
+function setErrorState(el, errorMsg) {
```

Comment:
Looks like setErrorState and setHidden might not be used anymore with the move to vue?